simulator/src/main/java/com/hazelcast/simulator/worker/metronome/SimpleMetronome.java

```diff
@@ -36,6 +36,21 @@ public static Metronome withFixedIntervalMs(int intervalMs) {
         return new SimpleMetronome(TimeUnit.MILLISECONDS.toNanos(intervalMs));
     }
 
+    /**
+     * Creates a {@link Metronome} instance with a fixed frequency in Hz.
+     *
+     * @param frequency frequency
+     * @return a {@link Metronome} instance
+     */
+    public static Metronome withFixedFrequency(float frequency) {
+        if (frequency == 0) {
+            return EMPTY_METRONOME;
```

Comment:
this feels a bit weird. that's not how I'd interpret `0 Hz`. it should be at least documented.